### PR TITLE
drop the 'ts' query param from google doc links

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner I believe this should do it

### CODE REVIEW

**Comment by andrewconner** [comment]
Looks like it would work. Let me know if you need help testing it.

**Comment by andrewconner** [comment]
Actually this should be able to statically test in a... test :)

**Comment by camhashemi** [comment]
wink wink

**Comment by camhashemi** [comment]
`PrenormalizerTest`, beautiful

File: kifi-backend/modules/shoebox/test/com/keepit/normalizer/PrenormalizerTest.scala [hunk 1]
```diff
@@ -151,12 +149,16 @@ class PrenormalizerTest extends Specification {
...
         "https://docs.google.com/document/d/1pFRKQtcZFqBYRdfcRbYT3TQaZaFqI1PgeOHEacF57q8/edit"
       prenormalize("https://docs.google.com/a/fuks.co.il/document/d/1Va2VsQwZqIgB73Eb0cWqPSF-bClwEBVCdgE3Nyik0sI/edit") ===
         "https://docs.google.com/document/d/1Va2VsQwZqIgB73Eb0cWqPSF-bClwEBVCdgE3Nyik0sI/edit"
       prenormalize("https://docs.google.com/a/42go.com/file/d/0B17Ux0DwquOwOVNWRGJLcEt6SW8/edit") === "https://docs.google.com/file/d/0B17Ux0DwquOwOVNWRGJLcEt6SW8/edit"
       prenormalize("https://docs.google.com/spreadsheet/ccc?authkey=CKqfiacO&hl=en_US&key=0AmrEm2VP6NfgdGU5czBIdnBYUHBYSE9wRzd6Q3VvakE&ndplr=1") === "https://docs.google.com/spreadsheet/ccc?authkey=CKqfiacO&key=0AmrEm2VP6NfgdGU5czBIdnBYUHBYSE9wRzd6Q3VvakE"
+      prenormalize("https://docs.google.com/document/d/1Gvv_9PzsqRoU_LaJo_MjdxxpgAggaGK-QZX6prnWVJU/edit?ts=56ba2849") === "https://docs.google.com/document/d/1Gvv_9PzsqRoU_LaJo_MjdxxpgAggaGK-QZX6prnWVJU/edit"
+      prenormalize("https://docs.google.com/document/d/1Gvv_9PzsqRoU_LaJo_MjdxxpgAggaGK-QZX6prnWVJU/edit?untouchable=true") === "https://docs.google.com/document/d/1Gvv_9PzsqRoU_LaJo_MjdxxpgAggaGK-QZX6prnWVJU/edit?untouchable=true"
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/test/com/keepit/normalizer/PrenormalizerTest.scala:161]: only taking out the `ts` param

**Pull request merged**